src/components/GeoSearchControls/__tests__/GeoSearchButton-test.tsx

```diff
@@ -16,9 +17,11 @@ describe('GeoSearchButton', () => {
     };
 
     const wrapper = shallow(
-      <GeoSearchButton {...props}>
-        Clear the current map refinement
-      </GeoSearchButton>
+      (
+        <GeoSearchButton {...props}>
+          Clear the current map refinement
+        </GeoSearchButton>
+      ) as ReactElementLike
```

Comment:
can you explain what `ReactElementLike` is about? can we also do it with other types without using prop-types?

Reply:
oh nvm. i just saw https://github.com/algolia/instantsearch.js/pull/4801/files